Bindings/Java/Matlab/Utilities/osimList.m

```diff
@@ -0,0 +1,95 @@
+classdef osimList
+properties
+        model % a reference to an opensim model
+        list; % an opensim list
+    end
+    
+    methods 
+        function obj = osimList(model,classname)
+            % Constructor for osimList class. Takes a model and a classname
+            % (string). constructs an instance of the osimList class with
+            % properties model and list
+            if nargin == 0
+                error('no inputs to constructor')
+            elseif nargin == 1
+                error('constructor takes two inputs, not one')
+            elseif nargin > 2
+                error(['2 inputs required, num2str(nargin) ' given])
```

Comment:
The `num2str(nargin)` is inside the string, and therefore will not be evaluated. I think you intended to place it outside the string.